components/sensor/ltr390.rst

```diff
@@ -21,10 +21,16 @@ The :ref:`I²C Bus <i2c>` is required to be set up in your configuration for thi
 
     sensor:
       - platform: ltr390
-        uv:
+        gain: "X18"
+        resolution: 20
```

Comment:
We try not to add config lines that are not required to let the user read the documentation and understand what they are able to change.
```suggestion
```